DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -205,6 +205,9 @@ private async Task FetchConfigAsyncWithTask()
 
                 logger.LogError(finalError.ErrorResponse.Message);
                 dvcEventArgs.Errors.Add(finalError);
+            } catch (System.ArgumentNullException e) {
```

Comment:
perhaps move the call to SetConfig to its own try...catch thats looking for this specific error, and comment what it means when its caught?